README.md

```diff
@@ -134,6 +134,10 @@ Client client = Clients.builder()
     .setClientId("{clientId}")
     .setScopes(new HashSet<>(Arrays.asList("okta.users.read", "okta.apps.read")))
     .setPrivateKey("/path/to/yourPrivateKey.pem")
+    // (or) .setPrivateKey("full PEM payload")
+    // (or) .setPrivateKey(Paths.get("/path/to/yourPrivateKey.pem"))
+    // (or) .setPrivateKey(inputStream)
```

Comment:
Supporting both `.setPrivateKey("full PEM payload")` & `.setPrivateKey(inputStream)` seems unnecessary to me. `.setPrivateKey("full PEM payload")` should be enough between the two I think.

Reply:
@arvindkrishnakumar-okta  Do you want to remove it from README.md only? Or you want to completely remove implementation starting from defining this method in the ClientBuilder interface?

Reply:
Implemented in here https://github.com/okta/okta-sdk-java/pull/498

Reply:
sorry, rethinking this I think we should retain `.setPrivateKey(inputStream)`.